### ISSUE TITLE
hydrops with predominance of subcutaneous edema

### ISSUE THREAD

**Issue by RiyaUthup** [obophenotype/human-phenotype-ontology#10835]
**Preferred term label:** hydrops with predominance of subcutaneous edema

**Synonyms**

**Definition (free text, please give PubMed ID)** 39191491

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@sparkste 
Is this a "bundled" term, i.e., we should probably use two HPO terms to describe it (hydrops AND subq edema), or is this recognized as a solitary entity?

**Comment by RiyaUthup** [comment]
In the reference article (https://pubmed.ncbi.nlm.nih.gov/39191491/), the authors use 'generalized hydrops with predominance of subcutaneous edema' as a bundled term. However, they also use 'subcutaneous edema' and 'hydrops' separately multiple times throughout the article.

**Comment by pnrobinson** [comment]
I think that this is the correct concept
[Fetal skin edema HP:0025672](https://hpo.jax.org/browse/term/HP:0025672)
It is not possible to capture all nuances of clinical observations using ontologies -- there are simply too many gradations and combinations of findings. The HPO approach tries to take one thing at a time, and so the correct way of coding this finding would be to use the terms for
Fetal skin edema HP:0025672
Hydrops fetalis HP:0001789

Thoughts? If you agree I will add Subcutaneous fetal edema as a synonym to the former term.

**Comment by sparkste** [comment]
Hi all,

Apologies that I missed this earlier.

Skin edema can be separate from hydrops, or it can be part of hydrops. I think it's fair to say a synonym for skin edema is subcutaneous fetal edema (but this is different from increased nuchal translucency or nuchal fold). Sometimes skin edema can be just around the head, often referred to as scalp edema. Diffuse skin edema is referred to as anasarca. So my vote is for skin edema and hydrops to be separate terms.

Thanks!
Teresa


Teresa Sparks, MD, MAS
Associate Professor, Maternal-Fetal Medicine and Clinical Genetics
Program Director, Maternal-Fetal Medicine Fellowship
Co-Director, Women's Reproductive Health Research/K12 Program
Division of Maternal-Fetal Medicine and Reproductive Genetics
University of California, San Francisco
________________________________

**Comment by RiyaUthup** [comment]
Thank you for the clarification. I agree that skin edema and hydrops should be considered separate terms.

**Comment by pnrobinson** [comment]
Thanks @sparkste and @RiyaUthup for the clarifications! I think we can close this now.
